src/ray/rpc/test/grpc_bench/README

```diff
@@ -0,0 +1,27 @@
+This package is used to benchmark gRPC's performance and overhead.
+
+To run the test, the instruction need to be followed in this:
+    https://github.com/Tradias/grpc_bench
+
+To run the test, a docker image need to be built and follow the instruction in grpc_bench to add new tests.
+
+-----------------------------------------------------------------------------------------------------------------------------------------
+| name                        |   req/s |   avg. latency |        90 % in |        95 % in |        99 % in | avg. cpu |   avg. memory |
```

Comment:
how should we generate this output? Is it manual?

Reply:
Yea, you should follow the grpc_bench and add new test sute there. Not very trivial for now.